src/Nest/_Generated/_Descriptors.generated.cs

```diff
@@ -6561,56 +6561,50 @@ public partial class GraphExploreDescriptor<T>  : RequestDescriptorBase<GraphExp
 	}
 	
 	///<summary>descriptor for XpackInfo <pre>Retrieve information about xpack, including build number/timestamp and license status</pre></summary>
-	public partial class XpackInfoDescriptor  : RequestDescriptorBase<XpackInfoDescriptor,XpackInfoRequestParameters, IXpackInfoRequest>, IXpackInfoRequest
+	public partial class XPackInfoDescriptor  : RequestDescriptorBase<XPackInfoDescriptor,XPackInfoRequestParameters, IXPackInfoRequest>, IXPackInfoRequest
```

Comment:
These change the casing of `Xpack` -> `XPack`. Should we keep the casing as is here for bwc and update in `master`?